isodatetime/main.py

```diff
@@ -285,7 +285,7 @@ def main():
         ],
     ]:
         arg_parser.add_argument(*o_args, **o_kwargs)
-    args = arg_parser.parse_args()
+    args = arg_parser.parse_intermixed_args()
```

Comment:
Thought you were going to do `if hasattr(arg_parse, 'parse_intermixed_args'):` ...

Reply:
yes, I was. I've been making my own life very complex with editing the version embedded in rose vs editing isodatetime.git. Looking forward to packaging!